### DISCUSSION TITLE
customize port names

### DISCUSSION THREAD

**Discussion by doug-oneal-nih** [opendcim/openDCIM#1480] [Q&A] [answered]
I’ve been tasked with defining an SOP for setting Connection values for each server. My problem is that my manager and our auditor will not accept port names that follow a strict pattern but want the names to be what the server uses. For example:

                Port 1 -> iLO
                Port 2 -> ens10f0np0
                Port 3 -> ens10f0np1
                Port 4 -> ib0
                Port 5 -> ib1

Some of our systems do not have an iLO but have a management port named something else. The ens* devices vary wildly across systems and even within the same server model the name is not consistent.  Most of the systems do not have InfiniBand devices. Is there a supported method for using arbitrary names in the ports field?

Currently using version 21.01 but we will be upgrading in the near future.

**Comment by wilpig** [comment]
A method other than just clicking the port and editing it to be whatever name you want?
<img width="1140" alt="Screenshot 2023-11-28 at 11 33 09 AM" src="https://github.com/opendcim/openDCIM/assets/1653882/a019c466-b83c-4c86-9d0a-53fc4b2db557">

**Reply by doug-oneal-nih** [reply]
I can select the port name but I cannot make any changes to it. I do have full privileges on my account.

![connections](https://github.com/opendcim/openDCIM/assets/81819179/cae9f109-4f88-482a-9a64-b14e03064f46)

**Reply by wilpig** [reply, marked as answer]
Click the 1
